LocalViewControl/LocalViewControl/CallViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  CallViewController.swift
+//  LocalViewControl
+//
+//  Created by Minhyuk Kim on 2021/06/11.
+//
+
+import UIKit
+import SendBirdCalls
+import Commons
+
+class CallViewController: UIViewController {
+    
+    var call: DirectCall!
+    
+    @IBOutlet weak var localVideoView: UIView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let remoteView = SendBirdVideoView(frame: self.view.frame)
+        view.embed(remoteView)
+        let localView = SendBirdVideoView(frame: localVideoView.frame)
+        localVideoView.embed(localView)
+        
+        call.updateRemoteVideoView(remoteView)
+        call.updateLocalVideoView(localView)
+        
+        call.delegate = self
+        
+        call.startVideo()
+        
+        self.mirrorLocalVideoView()
+    }
+    
+    @IBAction func flipCamera(_ sender: Any) {
+        self.call.switchCamera { error in
+            guard error == nil else {
+                print("Failed to switch video device \(error?.localizedDescription ?? "")")
+                return
+            }
+            
+            UIView.transition(with: self.localVideoView, duration: 1, options: .transitionFlipFromRight, animations: {
+                UIView.performWithoutAnimation {
```

Comment:
Self capturing is required.
Please add `[self]` (Info: you don't need to use weak self because it's a static method of UIView)

Reply:
Self capturing is automatically done when using `self.`. `[self]` is a syntactic sugar that removed the requirement to use `self.` when accessing `self`. So we don't need to add `[self]` here.